tests/tracing/static/collection-monitoring-false.output.yaml

```diff
@@ -19,24 +19,15 @@ data:
         endpoint: ${SUMO_ENDPOINT_DEFAULT_TRACES_SOURCE}
     extensions:
       health_check: {}
+      memory_ballast:
+        size_mib: 683
     processors:
       batch:
         send_batch_max_size: 512
         send_batch_size: 256
         timeout: 5s
       cascading_filter:
-        decision_wait: 30s
-        expected_new_traces_per_sec: 100
-        num_traces: 90000
-        policies:
-        - name: extended-duration
-          properties:
-            min_duration: 5s
-          spans_per_second: 500
-        - name: everything-else
-          spans_per_second: -1
-        probabilistic_filtering_ratio: 0.2
-        spans_per_second: 1660
+        num_traces: 200000
```

Comment:
That changes the default, doesn't it? Is that ok for the users? Are we planning to document this in the [CHANGELOG.md](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/CHANGELOG.md)/release notes?

Reply:
I don't think this is a concern since we never explained how to use prefilled values in `values.yaml` for `cascading_filter`. The initial idea was to provide instructions on how to just enable the processor in the pipeline, but this was never done and eventually we provide a template with instructions instead. So practically this might be considered a new thing. 

I am pretty sure we want to mention OTC update in the changelog though

Reply:
Agree on including it in the Changelog.

I understand the explanation but this still in some way is a breaking change 😓 for our users. It **was** possible to have this configured in the old yaml "schema" and now it would change with this PR being merged, right?

Reply:
I think it won't change much but of course we can mention this as well. So essentially something like:

* Update OpenTelemetry Collector version to v0.38.1-sumo
  * Move `insecure` parameter to separate configuration variable
  * Fix OTLP/HTTP metadata tagging
  * Update Cascading Filter to new version which adds filtering by number of errors and uses new config format
  * Change the default number of traces for Cascading Filter to 200000
  
  
@mat-rumian what do you think?

Reply:
If you could add an entry about this change in the [changelog](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/CHANGELOG.md) that would be great 👍

Reply:
Updated the changelog and put some links there